src/main/java/ro/jlg/staff/manager/infrastructure/EmployeeRepositoryImpl.java

```diff
@@ -0,0 +1,40 @@
+package ro.jlg.staff.manager.infrastructure;
+
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.stereotype.Repository;
+import ro.jlg.staff.manager.domain.Employee;
+import ro.jlg.staff.manager.domain.EmployeeRepository;
+
+import java.util.List;
+
+@Repository
+public class EmployeeRepositoryImpl implements EmployeeRepository {
+    private MongoTemplate mt;
```

Comment:
please rename to mongoTemplate. Abbrevations are fine if the name is very long, but this is not the case